docs/install/kubernetes.md

```diff
@@ -0,0 +1,225 @@
+
+Containers provisioned by Wave can be both disposable, i.e. ephemeral containers only accessible for a short period of time, and regular long-term registry-persisted container images.
+
+Waves build capabilities such rely on a specific integrations with Kubernetes and AWS EFS Storage making EKS & AWS a hard dependancy for a fully featuered deployment. 
+
+Wave also supports Augmentation only mode commonly reffered to as "wave-lite" which does not require these cloud provider specific integrations as such this instalation guide will initially focus on the installation on Generic Kubernetes in Augmentation only-mode which can be used across all Kubernetes distubutions
+
+see ../configuring-wave-build.md for details on how to extend your installation. 
+ 
+## Prerequesites
+
+An up to date kubernetes cluster
+postgres instance 
+Redis instance
+
+## Assumptions
+
+You have already deployed Seqera Platform
+You will be deploying the application into to the wave namespace
+you have appropriate permissions to perform these actions. 
+
+
+## Create namespace
+
+```
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "wave"
+  labels:
+    app: wave-app
+```
+
+## Configure Wave
+
+The following configuration example is a base please see the the wave configuration reference for all configuration options and examples. 
+
+Change the following values to match your environment. 
+
+database url 
+username 
+password
+
+redis password
+
+Seqera Platform API Endpoint commonly your platform instance followed by `/api` depending on your deployment you may have choosen to expose this on a subdomain such as `api.platforminstance.example`
+
+```
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: wave-cfg
+  namespace: "wave"
+  labels:
+    app: wave-cfg
+data:
+  config.yml:
+    wave:
+      build:
+        enabled: false
+      mirror:
+        enabled: false
+      scan:
+        enabled: false
+      blobCache:
+        enabled: false
+      db:
+        uri: "jdbc:postgresql://localhost:5432/wave"
+        user: "postgres"
+        password: "postgres"
+    redis:
+      uri: "rediss://REPLACE_ME_WAVE_REDIS_URL:6379"
+    tower:
+      endpoint:
+        url: REPLACE_ME_TOWER_SERVER_URL
+    micronaut:
+      executors:
+        stream-executor:
+          type: FIXED
+          number-of-threads: 16
+      netty:
+        event-loops:
+          default:
+            num-threads: 64
+          stream-pool:
+            executor: stream-executor
+      http:
+        services:
+          stream-client:
+            read-timeout: 30s
+            read-idle-timeout: 5m
+            event-loop-group: stream-pool
+    loggers:
+      env:
+        enabled: false
+      bean:
+        enabled: false
+      caches:
+        enabled: false
+      refresh:
+        enabled: false
+      loggers:
+        enabled: false
+      info:
+        enabled: false
+      metrics:
+        enabled: true
```

Comment:
Afaik this is required for prom metrics I think we can remove this from the install guide and move this into a section for configuring metrics.